src/main/java/uk/gov/hmcts/reform/sendletter/entity/EventType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.sendletter.entity;
+
+public enum EventType {
+    MANUALLY_MARKED_AS_CREATED,
+    MANUALLY_MARKED_AS_NOT_POSTED
```

Comment:
"MANUALLY_MARKED_AS_NOT_SENT" to match [LetterStatus.NotSent](https://github.com/hmcts/send-letter-service/blob/master/src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterStatus.java#L16)

Reply:
Of course